aten/src/ATen/quantized/Quantizer.h

```diff
@@ -149,12 +149,57 @@ struct CAFFE2_API PerChannelAffineQuantizer : public AffineQuantizer {
         axis() == other_per_channel_affine->axis();
   }
 
- private:
+ protected:
   Tensor scales_;
   Tensor zero_points_;
   const int64_t axis_;
 };
 
+/**
+ * PerChannelAffineFloatQParamsQuantizer is the same as PerChannelAffineQuantizer
+ * except that it expects both scale and zero point to be floating point values.
+ *
+ * This quantizer uses the kPerChannelAffineFloatQParams qscheme which is a variant of
+ * kPerChannelAffine.
+ *
+ * The quantize equation in this case looks like -
+ * Xq = (Xf - zero_point) * inv_scale, where inv_scale = 1.0/scale
```

Comment:
hmm, so just to confirm, this is still not the same `zero_point` conceptually that we use elsewhere?  I feel like people might get confused by this.  What would you think of naming it `bias` (as I think it was earlier in this PR), or `zero_point_caffe2`, etc?

Reply:
I think this is not a problem, we have different quantize functions as PerchannelAffineQuantizer